app/Post_Types/Diviner_Field/AdminModifications.php

```diff
@@ -76,9 +106,18 @@ function rc_scd_register_menu() {
 
 		add_menu_page( 'Diviner Fields', 'Manage	 Diviner Fields', 'manage_options', 'diviner-manage-fields', array( &$this,'rc_scd_create_dashboard'), 'dashicons-admin-generic', 30 );
 
+		add_submenu_page(
+			null,           // -> Set to null - will hide menu link
+			'Page Title',    // -> Page Title
+			'Menu Title',   // -> Title that would otherwise appear in the menu
+			'administrator', // -> Capability level
```

Comment:
this should be a capability, not a role name, e.g. `manage_options` or `install_plugins`